spec/requests/topics_controller_spec.rb

```diff
@@ -43,15 +43,16 @@
       expect(payload["post_stream"]["posts"].map { |p| p["id"] }).to eq([post.id, answer.id, answer_2.id, answer_3.id])
     end
 
-    it "includes QA comments for crawler view" do
-      get "/t/#{topic.slug}/#{topic.id}.html"
+    # temporarily disable crawler view test while we work on perf issues
+    # it "includes QA comments for crawler view" do
+    #   get "/t/#{topic.slug}/#{topic.id}.html"
```

Comment:
Could you use `pending` instead? Makes it a bit easier to remember that there is a spec that needs to be enabled in the future 🙂 
```
    it "includes QA comments for crawler view" do
      pending "temporarily disable crawler view test while we work on perf issues"
      get "/t/#{topic.slug}/#{topic.id}.html"
```

Reply:
Thanks for the tip. I've made the change. 👍🏼